prediction/src/algorithms/segment/src/data_generation.py

```diff
@@ -0,0 +1,78 @@
+import glob
+import os
+
+import numpy as np
+import pylidc as pl
+
+
+def get_dicom_paths(in_docker=True):
+    """Return DICOM paths to all LIDC directories depending on whether Docker is used or not
+    e.g. ['../images_full/LIDC-IDRI-0001/1.3.6.1.4.1.14519.5.2.1.6279.6001.298806137288633453246975630178/' \
+          '1.3.6.1.4.1.14519.5.2.1.6279.6001.179049373636438705059720603192']
+
+    Args:
+        in_docker: whether this method is invoked from within docker or from the prediction directory
+    """
+    if in_docker:
+        paths = glob.glob(os.path.join('/images_full', 'LIDC-IDRI-*', '**', '**'))
```

Comment:
Is `/images_full` correct here? Looks like an absolute path. Again, we should be using some kind of constant, not hardcoding these deep under ``prediction/src/algorithms/segment/src/data_generation.py`` :)

Reply:
In the docker container, the full LIDC images are mounted under /images_full/... . [test_correct_paths](https://github.com/concept-to-clinic/concept-to-clinic/pull/147/commits/8416d5895c38f9d1ed31a15f3668a3fecf804b75#diff-ed63136572f32bc5f7500b864f9bdfa9R9) should ensure that the folders exist. However, you're right - I'll add constants to our settings - or should I define them somewhere else?